pytorch_blade/tests/disc/ops/test_nn_ops.py

```diff
@@ -22,11 +23,14 @@ def _test_nn_ops(self, nn_ops_func, x=None):
             test_data = (test_data.to(self.device),)
         self._test_cvt_to_disc(nn_ops_func, test_data)
 
-    @skipTorchGE("1.12.0")
     def test_softmax(self):
         softmax = torch.nn.Softmax(dim=-1)
         self._test_nn_ops(softmax)
 
+        if utils.torch_version_number() >= utils.parse_version("1.12.0"):
```

Comment:
Nit： Is this update needed?

Reply:
Yes, i32 input suporting is not in this PR.